src/metpy/calc/kinematics.py

```diff
@@ -1056,3 +1057,40 @@ def q_vector(
     q2 = -mpconsts.Rd / (pressure * static_stability) * (dudy * dtempdx + dvdy * dtempdy)
 
     return q1.to_base_units(), q2.to_base_units()
+
+
+@exporter.export
+@add_grid_arguments_from_xarray
+@preprocess_and_wrap(broadcast=('height', 'u','latitude','potential_temperature'))
+@check_units('[temperature]', '[speed]','[length]','[dimensionless]')
+def eady_growth_rate(
+    potential_temperature,u,height,latitude,x_dim=-1, y_dim=-2,vertical_dim=-3
+):
+    r"""Calculate Eady growth rate (EGR) which is measure of baroclinic instability
+
+    .. math::  0.3098*g*abs(1/Coriolis parameter)*abs(du/dheight)/brunt_vaisala_frequency 
```

Comment:
I think this will render better as an equation
```suggestion
    .. math::  \frac{0.3098 g}{|f|} \frac{|du/dheight|}{N}

    where:

    * :math:`g` is the gravitational acceleration
    * :math:`f` is the Coriolis parameter
    * :math:`N` is the Brunt-Vaisala frequency
```